Create Blog Post “create-your-first-chrome-extension-in-javascript-to-hide-your-api-keys” (#1621)
Nexmo/deved-platform: content/blog/en/create-your-first-chrome-extension-in-javascript-to-hide-your-api-keys.md

---
title: Create Your First Chrome Extension in JavaScript to Hide Your API Keys
description: In this tutorial, you will learn how to create a chrome extension.

>   The example I will show today is to hide your Vonage API keys on the

Reviewer: ```suggestion
  The example we will walk through today is to hide your Vonage API keys on the
```

  dashboard. The knowledge you’ll acquire from following the steps of this
  tutorial will give you a foundation on how to create your own chrome
  extensions for different purposes.
author: amanda-cavallaro
published: true
published_at: 2021-08-23T10:21:09.589Z
updated_at: 2021-08-23T10:21:09.603Z
category: tutorial
tags:
  - chromeextension
  - javascript
  - browser
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---

> In this tutorial, you will learn how to create a chrome extension. The example I will show today is to hide your Vonage API keys on the dashboard. The knowledge you’ll acquire from following the steps of this tutorial will give you a foundation on how to create your own chrome extensions for different purposes.

Reviewer: ```suggestion
In this tutorial, you will learn how to create a chrome extension. The example we'll walk through today is to hide your Vonage API keys on the dashboard. The knowledge you’ll acquire from following the steps of this tutorial will give you a foundation on how to create your own chrome extensions for different purposes.
```

## What Is a Chrome Extension

A Chrome extension is a software program that allows you to extend the chrome functionalities and change your browser experience. An extension can be built with HTML, CSS, JavaScript and a `manifest.json` file. The latter one provides Chrome with the information required to run the scripts for the extension to work.

## Create the Chrome Extension Folder

Create a new folder in your computer that will contain the `manifest.json` file and the bootstrap file, in our example we will name it `index.js`. For my example, I'll name this folder `chrome-extension`.

## Create the `manifest.json` file

A `manifest.json` file is required when creating a chrome extension, as it tells your program the function to be executed. 

Create a `manifest.json` file. From your terminal on Unix, you can type `touch manifest.json`. 

Populate it with the required information as you can see in the code snippet below. 

> Let me explain each line step by step.

Reviewer: ```suggestion
Let's look at each line step by step.
```
